pkgs/development/python-modules/async_stagger/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytestCheckHook
+, pytest-asyncio
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "async_stagger";
+  version = "0.3.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "007l54fbk2dfzv3vmqz98m1i37mzxkkva5r4fiwq2pg8nb61fy0w";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-asyncio
+    pytest-mock
+  ];
+
+  disabledTests = lib.optional (!pythonOlder "3.8") [
```

this is of shape `[[string]]`
```suggestion
  disabledTests = lib.optionals (!pythonOlder "3.8") [
```

---

I always get those wrong. How do you remember which is which?

---

optionals -> "many" -> an array
optional   -> "one" -> singleton